apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -1,42 +1,355 @@
+            type={selectedProvider?.channel || ChannelTypeEnum.EMAIL}
+          />
+          <IntegrationEnvironmentPill
+            name={
+              environments?.find((environment) => environment._id === selectedProvider?.environmentId)?.name ||
+              'Development'
+            }
+          />
+        </Group>
+        <CenterDiv>
+          <When truthy={!haveAllCredentials}>
+            <WarningMessage spacing={12}>
+              <DisconnectGradient />
+              <div>
+                Set up credentials to start sending notifications
+                <br />
+                <a href={selectedProvider?.docReference} target="_blank" rel="noopener noreferrer">
+                  Explore set-up guide
+                </a>
+              </div>
+            </WarningMessage>
+          </When>
+          <Controller
+            control={control}
+            name="active"
+            render={({ field }) => (
+              <ActiveWrapper active={field.value}>
+                <Switch
+                  checked={field.value}
+                  label={field.value ? 'Active' : 'Disabled'}
+                  data-test-id="is_active_id"
+                  {...field}
+                />
+              </ActiveWrapper>
+            )}
+          />
+
+          <Controller
+            control={control}
+            name="identifier"
+            render={({ field }) => (
+              <InputWrapper>
+                <Input
+                  label="Instance key"
+                  error={errors.identifier?.message}
+                  rightSection={
+                    <CopyWrapper onClick={() => identifierClipboard.copy(field.value)}>
+                      {identifierClipboard.copied ? <Check /> : <Copy />}
+                    </CopyWrapper>
+                  }
+                  {...field}
+                />
+              </InputWrapper>
+            )}
+          />
+          {selectedProvider?.credentials.map((credential: IConfigCredentials) => (
+            <InputWrapper key={credential.key}>
+              <Controller
+                name={`credentials.${credential.key}`}
+                control={control}
+                render={({ field }) => <IntegrationInput credential={credential} errors={errors} field={field} />}
+              />
+            </InputWrapper>
+          ))}
+        </CenterDiv>
+        <Group mt={16} position="apart">
+          <Center inline>
+            <Text mr={5}>Explore our</Text>
+            <Text gradient>
+              <a href={selectedProvider?.docReference} target="_blank" rel="noopener noreferrer">
+                set-up guide
+              </a>
+            </Text>
+          </Center>
+          <Button disabled={!isDirty} submit={true}>
+            Update
+          </Button>
+        </Group>
+      </Form>
+    </SideBarWrapper>
+  );
 }
 
 const SideBarWrapper = styled.div<{ dark: boolean }>`
```

Comment:
Based on english use we should have had this component named `SidebarWrapper`. Oh well... technical debt.